src/MoBi.Presentation/Tasks/Edit/EditTaskFor.cs

```diff
@@ -216,5 +219,11 @@ protected virtual void InitializeSubPresenter(IPresenter subPresenter, IBuilding
             presenterWithFormulaCache.BuildingBlock = buildingBlock;
          }
       }
+
+      public List<IContainer> GetMoleculeProperties(IContainer container) =>
+         container?.Children
+            .OfType<IContainer>()
+            .Where(child => child.IsMoleculeProperties())
+            .ToList() ?? new List<IContainer>();
```

Comment:
`container?.Container(Constants.MOLECULE_PROPERTIES)`
would return the (only) Molecule Properties container (or **null** if it's not there)

Reply:
@benjaperez1983 he means this method could be simplified by using the short form he suggested.

Reply:
And I think it makes sense for the method to return only a single IContainer rather than a list. There will be at most one container called `MoleculeProperties`